helm/cluster-api-provider-azure/files/infrastructure/patches/versions/v1beta1/azuremachinetemplates.infrastructure.cluster.x-k8s.io.yaml

```diff
@@ -38,6 +38,13 @@
                     acceleratedNetworking:
                       description: AcceleratedNetworking enables or disables Azure accelerated networking. If omitted, it will be set based on whether the requested VMSize supports accelerated networking. If AcceleratedNetworking is set to true with a VMSize that does not support it, Azure will return an error.
                       type: boolean
+                    additionalCapabilities:
```

Comment:
New optional property. Giant Swarm vintage Azure clusters are not using AzureMachineTemplate CRs, so they are not affected by this change.